apps/api/src/utils/user_record.py

```diff
@@ -16,6 +16,7 @@ class Role(str, Enum):
     REVIEWER = "reviewer"
     ORGANIZER = "organizer"
     VOLUNTEER = "volunteer"
+    CHECKIN_LEAD = "checkin_lead"
```

Comment:
Issue: we'll need to update the authorization on the Admin site to allow the check-in lead to serve as an application manager.

Reply:
Sorry, and also the applicants summary via `ADMIN_ROLES`

Reply:
I added `checkin_lead` directly to `ADMIN_ROLES` in [`46f9c93`](https://github.com/HackAtUCI/irvinehacks-site-2024/pull/323/commits/46f9c936668630a8d071b632c8e449b7bdda9a1e)